.eslintrc.json

```diff
@@ -7,10 +7,12 @@
 		"Cosmoz": false,
 		"CustomElements": false,
 		"HTMLImports": false,
-		"moment": false,
 		"Polymer": false,
 		"WeakMap": false
 	},
+	"parserOptions": {
```

Comment:
We can actually do `"env": { "es6": true }` to also support the new types such as `Set`.

Reply:
Added that.